src/detectors/builtin/sendInLoop.ts

```diff
@@ -103,4 +181,13 @@ export class SendInLoop extends ASTDetector {
       stmt.kind === "statement_foreach"
     );
   }
+
+  private getCalleeName(expr: AstExpression): string | undefined {
```

Comment:
Why don't you use https://github.com/nowarp/misti/blob/e1fe0238cb7da1dcd616e8b75b81bc0e810ccd19/src/internals/ir/callGraph.ts#L168? I think we should generalize its functionality creating a function that gets the name from the function call.